### DISCUSSION TITLE
websocket报错403

### DISCUSSION THREAD

**Discussion by tyh1820732766** [nonebot/discussions#71] [问答]
启用了gocqhttp的反向websocket之后，无法与nonebot进行对接，日志报错
[INFO] uvicorn | ('127.0.0.1', 60276) - "WebSocket /ws/" 403
[INFO] websockets | connection failed (403 Forbidden)
[INFO] websockets | connection closed

**Comment by yanyongyu** [comment]
请问你的nonebot版本是？

**Reply by tyh1820732766** [reply]
1.9.1
